modules/eventing/pages/eventing-handler-keepLastN_661.adoc

```diff
@@ -0,0 +1,240 @@
+= Function: Keep the Last N User Items 6.6.1
```

Comment:
Rename the file to remove the version number (eventing-handler-advanced-keepLastN.adoc)

Reply:
There are two versions (one will work up through 6.6.0 and the other will work starting from 6.6.1 on)
eventing-handler-keepLastN.adoc
eventing-handler-keepLastN_661.adoc
I have no issues with renaming - but we still need the two versions since this is a 6.6.0 document set.